csharp/Profiler/LineProfile.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+
+namespace Profiler
+{
+    /// <summary>
+    /// Represents a profiled line.
+    /// </summary>
+    public class LineProfile
+    {
+        /// <summary>
+        /// Percent of total duration used by line.
+        /// </summary>
+        public double Percent { get; set; } = 0.00;
```

Comment:
Would use decimal here, but it's just a nitpick, if the current code uses double it should be totally fine for this purpose as well. But if we add self percent we should use decimal because we will test that all lines add up to 100%, and double might have bigger error margin.

Reply:
Will decimal vs double matter considering how we already round the value prior to setting it?

Either way, decimal is fine. 👍 I just chose the "cheapest" one tbh., not that it matters much at this scale.

Reply:
Let's keep it.